src/main/java/org/owasp/esapi/reference/validation/HTMLValidationRule.java

```diff
@@ -105,20 +105,27 @@ private String invokeAntiSamy( String context, String input ) throws ValidationE
 			}
 			throw new ValidationException( context + " is required", "AntiSamy validation error: context=" + context + ", input=" + input, context );
 	    }
-	    
+
 		String canonical = super.getValid( context, input );
 
 		try {
 			AntiSamy as = new AntiSamy();
 			CleanResults test = as.scan(canonical, antiSamyPolicy);
-			
+
 			List<String> errors = test.getErrorMessages();
 			if ( !errors.isEmpty() ) {
-				LOGGER.info( Logger.SECURITY_FAILURE, "Cleaned up invalid HTML input: " + errors );
+				StringBuilder sb = new StringBuilder();
```

Comment:
@ZiluckMichael I don't believe the StringBuilder is necessary.  I think you can use the toString value of the list as-is.

`throw new ValidationException( context + ": Invalid HTML input", "Invalid HTML input does not follow rules in antisamy-esapi.xml: context=" + context + " errors=" + errors.toString());
`
I had to test it locally with a base ArrayList, but it did what I was expecting:
```
List<String> values = new ArrayList<>();
        values.add("one");
        values.add("two");
        values.add("three");
        
        System.out.println(values.toString());

```
results in:

> [one, two, three]

If you could try it without the StringBuilder and verify the additional iteration isn't needed I'd certainly appreciate it.

Reply:
I agree with @jeremiahjstacey here about not needing StringBuilder here. This in fact is a pretty common pattern throughout ESAPI and I'd prefer to not make this occurrence different than the others, all which already seem to work and do the right thing.

Reply:
I was always under the impression that `.toString()` for lists in Java printed out the hexed hash code, but TIL! Sounds good. Fixed and pushed :)